claves.txt

```diff
@@ -0,0 +1 @@
+miclave
```

Comment:
se ve bien! pero podrias separar "miclave" como "mi clave" ?

Reply:
woriking on it!